styles.css

```diff
@@ -251,3 +280,193 @@ textarea {
   background-color: #2230d2;
   color: #fff;
 }
+
+@media (min-width: 600px) {
+  header {
+    display: flex;
+    justify-content: space-between;
+  }
+
+  #nav-links {
+    display: flex;
+    flex-direction: row;
+    justify-content: space-evenly;
+    width: 28vw;
+    margin-right: 20px;
+  }
+
+  .logo {
+    margin-left: 53px;
+  }
+
+  .hamburger {
+    display: none;
+  }
+
+  .intro {
+    display: flex;
+    flex-direction: column;
+    width: min-content;
+    margin: 10em 0 0 10em;
+  }
+
+  .ul {
+    display: flex;
+    width: 14vw;
+    list-style: none;
+  }
+
+  .heading #glad {
+    display: inline;
+  }
+
+  .paragraph {
+    width: 920px;
+    height: 98px;
+    font-size: 20px;
+    line-height: 28px;
+    text-align: left;
+  }
+
+  h2 {
+    font-size: 20px;
+    line-height: 28px;
+    text-align: left;
+  }
+
+  .data {
+    display: flex;
+    flex-direction: row;
+    width: 20vw;
+  }
+
+  .canopy {
+    width: fit-content;
+    height: 24px;
+    font-weight: 500;
+    font-size: 18px;
+    line-height: 24px;
+    color: #344563;
+  }
+
+  #projects-section {
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+    padding: 50px 0 150px 0;
+  }
+
+  .projects {
+    display: grid;
+    background: #fff;
+    max-width: 1156px;
+    border: 2px solid #a7aeff;
+    border-radius: 24px;
+    grid-template-columns: min-content;
+    margin: 70px 20px 70px 20px;
+  }
+
+  .content {
+    margin: 40px 0 0 5px;
+    box-sizing: border-box;
+  }
+
+  .projects .content {
+    grid-column-start: 2;
+  }
+
+  .projects .big-btn {
+    grid-column-start: 2;
+    width: fit-content;
+  }
+
+  .swap {
+    display: flex;
+    flex-direction: row-reverse;
+    background: #fff;
+    border: 2px solid #a7aeff;
+    border-radius: 24px;
+    margin: 70px 20px 70px 20px;
+  }
+
+  .swap .content {
+    margin-top: 50px;
+    width: 43vw;
+  }
+
+  .swap .header {
+    margin-right: 20px;
+  }
+
+  .img {
+    width: 544px;
+    height: 448px;
+    margin: 20px 0 20px 20px;
+  }
+
+  .content p {
+    width: 45vw;
+    text-align: left;
+    font-size: 16px;
+  }
+
+  .About {
+    display: flex;
+    justify-content: space-between;
+    background: white;
+    width: 100vw;
+    padding: 100px 60px;
+    border-top-right-radius: 100px;
+    position: relative;
+    top: -120px;
+  }
```

Comment:
- [ ] Please make sure the mobile version doesn't change when coding the desktop one. Currently, the mobile version is stuck to the left side of the screen, kindly fix that.

![](https://i.imgur.com/Q7WTknb.png)